demos/slack-alert-app/slackbot.py

```diff
@@ -0,0 +1,116 @@
+import time
+import logging
+import os
```

Comment:
you should add a requirements.txt file that includes 
`schedule`, `slack`, and `pxapi`.

```
pip freeze
```
and copy the versions for slack, schedule, and pxapi.